ticketutil/servicenow.py

```diff
@@ -0,0 +1,321 @@
+"""
+ServiceNow ticket
+
+Provided, you have user and pwd, you can create new ticket like this,
+but you should need other attributes like Category, Item and other non-optional
+attributes from the ServiceNow web GUI.
+
+TODO:
+api PATCH method would enable editing comment
+api DELETE method - maybe, not that necessary
+"""
+
+
+import logging
+import os
+
+import requests
+
+from . import ticket
+
+# TODO ask dranck about credits
+__author__ = 'dranck, rnester, kshirsal, pzubaty'
+
+# Disable warnings for requests because we aren't doing certificate
+# verification
+requests.packages.urllib3.disable_warnings()
+
+DEBUG = os.environ.get('TICKETUTIL_DEBUG', 'False')
+
+if DEBUG == 'True':
+    logging.basicConfig(level=logging.DEBUG)
+else:
+    logging.basicConfig(level=logging.INFO)
+
+
+class ServiceNowTicket(ticket.Ticket):
```

Comment:
Where are the unit tests? You should definitely include some even though the project has yet to start any.